config/locales/en.yml

```diff
@@ -29,6 +29,7 @@ en:
       am: am
       formats:
         default: "%Y-%m-%d %I:%M%p"
+        event: "%Y-%m-%d"
```

Comment:
It looks like Rails has a couple built-in date formats that are identical: `db` and `inspect`. Should we use a built-in or is `event` more descriptive?

Reply:
Date formats are cheap, so I buy that we want this in our domain language.